deploy/configs/applications/modules.yaml

```diff
@@ -36,6 +36,7 @@ modules:
       - py-bluepymm
       - py-bluepyopt
       - py-efel
+      - py-pytouchreader
```

Comment:
If this is used as module by users (or you) then its ok to expose.

Reply:
It'll be used by @bbpbuildbot :)